frontend/packages/knative-plugin/src/components/overview/KSRoutesOverviewList.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { K8sResourceKind } from '@console/internal/module/k8s';
+import { SidebarSectionHeading } from '@console/internal/components/utils';
+import { getKnativeRoutesLinks } from '../../utils/resource-overview-utils';
+import KSRoutesOverviewListItem from './KSRoutesOverviewListItem';
+import KSRouteSplitListItem from './KSRouteSplitListItem';
+
+export type KSRoutesOverviewListProps = {
+  ksroutes: K8sResourceKind[];
+  resource: K8sResourceKind;
+};
+
+const KSRoutesOverviewList: React.FC<KSRoutesOverviewListProps> = ({ ksroutes, resource }) => (
+  <>
+    <SidebarSectionHeading text="Routes" />
+    {_.isEmpty(ksroutes) ? (
+      <span className="text-muted">No Routes found for this resource.</span>
+    ) : (
+      <ul className="list-group">
+        {ksroutes.map((route) => (
+          <>
+            <KSRoutesOverviewListItem ksroute={route} />
+            {getKnativeRoutesLinks(route, resource).map((splitRoute) => (
+              <KSRouteSplitListItem route={splitRoute} />
+            ))}
+          </>
+        ))}
```

Comment:
create a re-useable component named `KSRoutes`(may be) and
```suggestion
{ksroutes.map((route) => {
          const routeLinks = getKnativeRoutesLinks(route, resource);
          return <KSRoutes key={route.metadata.uid} route={route} routeLinks={routeLinks} />;
        })}
```

Reply:
`export const KSRoutes: React.FC<KSRoutesProps> = ({ route, routeLinks }) => (
  <>
    <KSRoutesOverviewListItem ksroute={route} />
    {routeLinks.map((splitRoute) => (
      <KSRouteSplitListItem key={splitRoute.uid} route={splitRoute} />
    ))}
  </>
);`

Reply:
Thanks. Updated.